src/vehicle_display.cpp

```diff
@@ -478,3 +478,35 @@ void vehicle::print_fuel_indicator( const catacurses::window &win, const point &
         }
     }
 }
+
+void vehicle::print_speed_gauge( const catacurses::window &win, const point &p, int spacing )
+{
+    if( spacing < 0 ) {
+        throw std::invalid_argument( "received negative value for spacing" );
+    }
+    if( !cruise_on ) {
+        return;
+    }
+    // target speed < current speed
```

Comment:
This comment could mislead people into thinking it's saying something about target speed being lower than current speed.